udata/cgu.md

```diff
@@ -0,0 +1,6 @@
+# Generic terms and conditions
```

Comment:
`cgu.md` is based on a french naming (Conditions Générales d'Utilisation). Would not it be more consistent to name it `terms.md`?

Also, should we rather return a 404 page on the terms page if nothing has been specified? I'd be more comfortable to return nothing rather than a content which cannot be used as is.

Something like `udata doctor` could be used to list recommended changes to improve udata settings beyond its defaults.

Reply:
👍 for `terms.md`. Not sure about the `404`.